test/OptionsTest.hs

```diff
@@ -0,0 +1,80 @@
+module Main (main) where
+
+import           Data.Version                   (showVersion)
+import           System.Process                 (readProcess)
+
+import           Test.Framework                 (defaultMain)
+import           Test.Framework.Providers.HUnit (hUnitTestToTests)
+import           Test.HUnit
+
+import           Language.Egison                (version)
+
+main :: IO ()
+main = defaultMain . hUnitTestToTests . test $ TestList
+    [ TestLabel "load-file option" . TestCase $ assertEgisonCmd
+        (interpreter "1\n")
+        ["--load-file", "test/fixture/name-of-file-to-load.egi"]
```

Comment:
In [this example in the docs](https://egison.readthedocs.io/en/latest/reference/command-line-options.html#l-load-file), the name `name-of-file-to-load.egi` and `name-of-file-to-test.egi` is meant to be replaced with some other names, and using these names in the tests feels a bit strange.
Can you rename it? (`a.egi`, for example?)

Reply:
rename to `a.egi` `b.egi`